### PR TITLE
Fix outdated use of `File` in `gdscript_basics`

### PR DESCRIPTION
**Pull request by AThousandShips**
* Fixes: #7665

### CODE REVIEW

File: tutorials/scripting/gdscript/gdscript_basics.rst [hunk 1]
```diff
@@ -1865,9 +1865,9 @@ Here is an example:
     var my_file_ref
 
     func _ready():
-        var f = File.new()
+        var f = FileAccess.new()
```
Comment (Piralein) [inline tutorials/scripting/gdscript/gdscript_basics.rst:1868]: FileAccess can't be directly instantiated, using something like `FileAccess.open("user://example_file.json", FileAccess.READ)` returns a FileAccess reference.

**Review by Piralein — CHANGES_REQUESTED** [review]

File: tutorials/scripting/gdscript/gdscript_basics.rst (hunk 1 above)
Comment (AThousandShips) [reply tutorials/scripting/gdscript/gdscript_basics.rst]: Done!

**Review by Piralein — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Thank you! 🎉 Merged.

**Comment by AThousandShips** [comment]
Thank you! Nice to do some documentation work over here too and not just for the classref

**Pull request merged**